.github/workflows/send-system-notification.yml

```diff
@@ -51,6 +54,10 @@ jobs:
             slack_token_secret_name="$(echo "${channel_config}" | jq -r ".slack_token_secret_name")"
             echo "Slack token secret name: ${slack_token_secret_name}"
             echo "SLACK_TOKEN_SECRET_NAME=${slack_token_secret_name}" >> "$GITHUB_ENV"
+
+            # Convert Markdown links in message [text](url) to Slack format <url|text>
+            echo "Convert message from Markdown to Slack format"
+            echo "SLACK_MESSAGE=$(echo "${{ inputs.message }}" | sed -E 's/\[(.+)\]\((.+)\)/<\2|\1>/g')" >> "$GITHUB_ENV"
```

Comment:
I would expect to see this pipe sed bit down lower, on the `payload: | ... text: ` line. I assume there was some constraint that make you want to pipe it into an environment variable instead?

Reply:
Oh, its that the content of the `payload` value aren't bash. I'm just gonna leave this comment here in case it illuminates anyone else.